net-dns/knot/knot-1.6.8.ebuild

```diff
@@ -53,40 +57,28 @@ src_configure() {
 
 src_compile() {
 	default
-	use doc && emake -C doc html-local singlehtml pdf-local info-local
+	use doc && emake -C doc html
 }
 
-# Portage's automatic test code runs "make -n check" to see if there
-# is a "check" target, but that fails here because the test
-# infrastructure hasn't been built yet. Just run "make check", which
-# will build it and then run the tests.
 src_test() {
-	emake -j1 check
+	emake check
 }
 
 src_install() {
 	default
 	keepdir /var/lib/${PN}
 
 	if use doc; then
-		dodoc doc/_build/latex/KnotDNS.pdf
-
 		docinto html
 		dodoc doc/_build/html/*.html doc/_build/html/*.js
 		docinto html/_sources
 		dodoc doc/_build/html/_sources/*
 		docinto html/_static
 		dodoc doc/_build/html/_static/*
```

Comment:
and put it before the default, then you don't even need `einstalldocs`, as that is implicitly called by `default_src_install` in `EAPI=6`